src/act/node/candid/service/method.rs

```diff
@@ -3,7 +3,7 @@ use quote::{format_ident, quote};
 
 use crate::{
     act::{
-        node::{CandidType, Context, Param, ReturnType},
+        node::{candid::func::Mode, CandidType, Context, Param, ReturnType},
```

Comment:
If we are going to use func::Mode in here as well then we should probably move it to a neutral location. Maybe node_parts? I'm not sure